deployment/RS01SBPS_Deploy.csv

```diff
@@ -1,91 +1,101 @@
-RR1713,,,,RS01SBPS-SF01A-4B-VELPTD102,4,1,2017-08-04T19:30:00,2018-06-24T02:00:00,ATAPL-68870-001-0143,,ATAPL-70114-00006,44.52897,-125.38966,,,2906,
-RR1713,,,,RS01SBPS-SF01A-4F-PCO2WA101,4,1,2017-08-04T19:30:00,2018-06-24T02:00:00,ATAPL-68870-001-0143,,ATAPL-58336-00004,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-PC01A-05-ADCPTD102,5,1,2018-06-24T02:00:00,,ATAPL-69839-001-0101,,ATAPL-58315-00003,44.52897,-125.38966,,195,2906,
-RR1809,,,,RS01SBPS-PC01A-06-VADCPA101,5,1,2018-06-24T02:00:00,,ATAPL-69839-001-0101,,ATAPL-58345-00003,44.52897,-125.38966,,195,2906,
-RR1809,,,,RS01SBPS-PC01A-07-CAMDSC102,5,1,2018-06-24T02:00:00,,ATAPL-69839-001-0101,,ATAPL-58317-00009,44.52897,-125.38966,,195,2906,
-RR1809,,,,RS01SBPS-PC01A-08-HYDBBA103,5,1,2018-06-24T02:00:00,,ATAPL-69839-001-0101,,ATAPL-58324-00005,44.52897,-125.38966,,195,2906,
-RR1809,,,,RS01SBPS-PC01A-4A-CTDPFA103,5,1,2018-06-24T02:00:00,,ATAPL-69839-001-0101,,ATAPL-66662-00001,44.52897,-125.38966,,195,2906,
-RR1809,,,,RS01SBPS-PC01A-4A-DOSTAD103,5,1,2018-06-24T02:00:00,,ATAPL-69839-001-0101,,ATAPL-58320-00005,44.52897,-125.38966,,195,2906,
-RR1809,,,,RS01SBPS-PC01A-4B-PHSENA102,5,1,2018-06-24T02:00:00,,ATAPL-69839-001-0101,,ATAPL-58337-00014,44.52897,-125.38966,,195,2906,
-RR1809,,,,RS01SBPS-PC01A-4C-FLORDD103,5,1,2018-06-24T02:00:00,,ATAPL-69839-001-0101,,ATAPL-58322-00005,44.52897,-125.38966,,195,2906,
-RR1809,,,,RS01SBPS-SF01A-2A-CTDPFA102,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-66662-00002,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-SF01A-2A-DOFSTA102,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-58694-00001,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-SF01A-2D-PHSENA101,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-58337-00013,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-SF01A-3A-FLORTD101,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-58322-00001,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-SF01A-3B-OPTAAD101,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-58332-00006,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-SF01A-3C-PARADA101,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-66645-00002,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-SF01A-3D-SPKIRA101,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-58341-00002,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-SF01A-4A-NUTNRA101,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-68020-00004,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-SF01A-4B-VELPTD102,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-70114-00002,44.52897,-125.38966,,,2906,
-RR1809,,,,RS01SBPS-SF01A-4F-PCO2WA101,5,1,2018-06-24T02:00:00,,ATAPL-68870-001-0140,,ATAPL-58336-00002,44.52897,-125.38966,,,2906, 
\ No newline at end of file
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
```

Comment:
please revert line endings

Reply:
Back to CRLF?

Reply:
If you compare to the original in GitHub ("View all changes") you can see every line is highlighted for change.